lib/file_upload_restrictions/files_controller_overrides.rb

```diff
@@ -5,7 +5,7 @@ module FilesControllerOverrides
       def api_create
         @policy, @attachment = Attachment.decode_policy(params[:Policy], params[:Signature])
         @context = @attachment.context
-        size_limit = @context&.account ? @context&.account.max_file_size[:value].to_f : nil
+        size_limit = @context&.account && !(@context.account.max_file_size[:value].nil?) ? @context&.account.max_file_size[:value].to_f : nil
```

Comment:
I think `@context.account.max_file_size[:value].present?` would suffice here? Or am I missing a case where !nil? is the more appropriate?

Actually, I think doing this would work equally as well?

```suggestion
        size_limit = @context&.account && @context&.account.max_file_size[:value]&.to_f
```

This way it will return nil if the value is nil otherwise a float. Will that work?

Reply:
This does work equally well, and has been changed.